src/garrysmod/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM quay.io/rbreslow/garrysmodds:slim
 
-# Download Counter-Strike: Source DS with steamcmd for content dependencies
+# Download Counter-Strike: Source DS
 RUN set -ex \
     && /usr/games/steamcmd +login anonymous \
     +app_update 232330 \
     validate \
     +quit
+
+COPY cfg/mount.cfg garrysmod/cfg/mount.cfg
```

Comment:
This doesn't need to be a volume mount in Docker Compose since it never changes.